.github/workflows/azure-ci.yml

```diff
@@ -49,6 +47,10 @@ jobs:
           stack-version: ${{ env.ELK_VERSION }}
           security-enabled: false
 
+      - name: Elasticsearch disable dynamic mapping for findings
+        run: |
+          curl -s -XPUT http://localhost:9200/_index_template/common_template -H 'Content-Type: application/json' -d '{ "index_patterns": ["logs-cloud_security_posture.findings*"], "priority": 500, "template": { "mappings": { "dynamic": false, "properties": { "resource": { "dynamic": false, "properties": { "type": { "type": "keyword" } } }, "result": { "dynamic": false, "properties": { "evaluation": { "type": "keyword" } } }, "rule": { "dynamic": false, "properties": { "benchmark": { "dynamic": false, "properties": { "id": { "type": "keyword" } } }, "tags": { "type": "keyword" } } }, "agent": { "dynamic": false, "properties": { "id": { "type": "keyword" }, "version": { "type": "keyword" } } }, "@timestamp": { "type": "date" } } } } }'
+
```

Comment:
@jeniawhite Why is it necessary to disable dynamic mapping for findings?

Reply:
If we do not disable dynamic mapping this creates a scenario when we drop events with more than 1000 fields.
In our case since the raw resource data is part of the document, we essentially do not send evaluations for large resources.
This creates flakiness in the test cases.